src/web_app/html/index_template.html

```diff
@@ -28,6 +28,8 @@
   <link rel="canonical" href="{{ room_link }}">
 {% endif %}
   <link rel="stylesheet" href="/css/main.css">
+  <!-- callstats dependencies: http://www.callstats.io/api/#integrating-with-your-sdk -->
+  <script src="https://api.callstats.io/static/callstats.min.js"></script>
```

Comment:
Just make sure to pull callstats.js from periodically, so that the latest version of callstats.js is in use.